web/client/translations/data.it-IT.json

```
@@ -1678,7 +1678,16 @@
                 "syncOnMap": "Sincronizza la mappa con i filtri",
                 "enableTimeSync": "Sincronizza con l'istante o l'intervallo temporale selezionato",
                 "disableTimeSync": "Non sincronizzare con l'istante o l'intervallo temporale selezionato",
-                "createNewChart": "crea un grafico per il livello selezionato"
+                "createNewChart": "crea un grafico per il livello selezionato",
+                "enableSnapping": "Attivare lo scatto",
+                "disableSnapping": "Disattivare lo snap",
+                "snapToLayer": "Aggancia al livello",
+                "snappingSettings": {
+                    "header": "Impostazioni",
+                    "vertex": "Vertice",
+                    "edge": "Bordo",
+                    "pixelTolerance": "Tolleranza pixel"
+                  }
```

- [ ] This is for italian.
```suggestion
                "enableSnapping": "Attiva snapping",
                "disableSnapping": "Disattiva snapping",
                "snapToLayer": "Aggancia al livello",
                "snappingSettings": {
                    "header": "Impostazioni",
                    "vertex": "Vertice",
                    "edge": "Bordo",
                    "pixelTolerance": "Tolleranza (pixel)"
                  }
```